src/Framework/Setup/AbstractSetupGacela.php

```diff
@@ -4,50 +4,11 @@
 
 namespace Gacela\Framework\Setup;
 
-use Gacela\Framework\Config\GacelaConfigBuilder\ConfigBuilder;
-use Gacela\Framework\Config\GacelaConfigBuilder\MappingInterfacesBuilder;
-use Gacela\Framework\Config\GacelaConfigBuilder\SuffixTypesBuilder;
+use Gacela\Framework\Bootstrap\AbstractSetupGacela as BootstrapAbstractSetupGacela;
 
-abstract class AbstractSetupGacela implements SetupGacelaInterface
+/**
+ * @deprecated in favor of \Gacela\Framework\Bootstrap\AbstractSetupGacela
```

Comment:
What about defining the version we will remove the deprecation class/method in the PhpDoc itself?

Reply:
I would prefer don't. The "deprecated" tag already tell you that you shouldn't use it and it will be removed in a future version. So it's up to the dev to avoid using the deprecated and use the new ones.